### PR TITLE
Update branding for preview4

### PR DESCRIPTION
**Pull request by wtgodbe**
CC @dagood @danmosemsft @mmitche @leecow

### CODE REVIEW

**Comment by dagood** [comment]
So `master` will produce `3.0.0-preview4`, `3.0.0-preview5`, etc., and `release/3.0` will produce `3.0.0-preview`? Is it ok that nuget versions will have a weird sort? (`3.0.0-preview4` will be later than `3.0.0-preview`, even when `release/3.0` is building for preview 5?)

**Comment by wtgodbe** [comment]
@dagood Currently release/3.0 is building `preview3`

File: BranchInfo.props [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
```
Comment (dagood) [inline BranchInfo.props]: Same issue with `PreReleaseLabel` a few lines up.

File: BranchInfo.props [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
 
     <ReleaseSuffix>$(PreReleaseLabel)</ReleaseSuffix>
-    <ReleaseBrandSuffix>Preview</ReleaseBrandSuffix>
+    <ReleaseBrandSuffix>Preview4</ReleaseBrandSuffix>
```
Comment (dagood) [inline BranchInfo.props]: This is also missing a space.

Comment (dagood) [reply BranchInfo.props]: I wouldn't be opposed to using MSBuild property functions to lowercase and remove the space to keep brand and prerelease label in sync.

**Comment by wtgodbe** [comment]
My bad, it should be fixed now. Will do something similar in Master

**Pull request merged**